build/commands/lib/logging.js

```diff
@@ -124,6 +124,26 @@ function command (dir, cmd, args) {
   status(`${cmdArrowStyle('>')} ${cmdCmdStyle(cmd)} ${args.join(' ')}`, true)
 }
 
+function printLogFailuresToJson(allPatchStatus, bravePath) {
+  const failedPatches = allPatchStatus.filter((patch) => patch.error)
+  if (!failedPatches.length) {
+    return;
+  }
+
+  const patchFailuresOutput = failedPatches.map(({path, patchPath}) => {
+    return {
+      // Trimming the patch path to be relative to the brave core repo.
+      patchPath:
+        patchPath.replace(bravePath+'/', ''),
```

Comment:
nit: might not be `/` on windows? But you might not care either...

Reply:
Done